velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -698,14 +698,15 @@ void ParquetRowReader::filterRowGroups() {
     auto rowGroupInRange =
         (fileOffset >= options_.getOffset() &&
          fileOffset < options_.getLimit());
+
+    auto isExcluded =
```

Comment:
nit: for readability, merge this two in a single code block and add blank line afterwards

```
auto isExcluded = (i < res.totalCount && bits::isBitSet(res.filterResult.data(), i));
auto isEmpty = rowGroups_[i].num_rows == 0;

if (...) {
```

Reply:
The first statement rollovers to next line by `format-fix`